eHealthAfrica/aether
# Aet ui load mappings from kernel

Fetch mapping data from the `kernel` and transform and store as `ui pipelines`
NOTE: 

- Pipeline input data and schema defaults to `{}`
- Subsequent mapping updates from the kernel are not effected to prevent overwriting changes on the ui.

#### How to test

- Setup kernel with project, schema, project schema and mappings
- Load ui
- A pipeline should be added to the pipeline list reflecting the mappings on kernel
